.jenkins/common.groovy

```diff
@@ -42,7 +42,7 @@ def runTestCommand (platform, project, gfilter, boolean rocmExamples=false, Stri
     if (platform.jenkinsLabel.contains('gfx90a'))
     {
         hmmTestCommand = """
-                            HSA_XNACK=0 GTEST_LISTENER=NO_PASS_LINE_IN_LOG  ./rocsparse-test  --rocsparse-enable-debug --rocsparse-clients-enable-test-debug-arguments --gtest_output=xml:test_detail_hmm_xnack_off.xml --gtest_color=yes --gtest_filter=${gfilter}-*known_bug*
```

Comment:
Again I am probably missing something, but don't you still want to have `--rocsparse-clients-enable-test-debug-arguments` so that the signature arguments are checked?

Reply:
It is enabled by default in the main of rocsparse-test.